### DISCUSSION TITLE
How do I get the current topic name?

### DISCUSSION THREAD

**Discussion by ScotterMonk** [python-telegram-bot/python-telegram-bot#4302] [Q&A]
I read update.message.chat.title for chat group name.
But how do I get the topic name?
Thanks!

**Comment by Poolitzer** [comment]
You can not get it. You need to watch for the forum created service messages. Messages sent in a forum without a reply to message will reply to that service message.

**Reply by ScotterMonk** [reply]
Thanks for the info! I'm confused. You say it is not obtainable BUT then you say "watch for the forum created service messages." Please know that I'm a newb and wondering if you are saying there IS a way to do this? If so, can you give a bit more direction as to how I would "watch for (and parse) the forum created service messages"?

**Reply by Poolitzer** [reply]
You can not call a get function to get them.

If your bot is in a group with forum enabled, you get service messages as message updates when something happens to a topic (created, edited, closed, etc.) so you can setup a MessageHandler with the appropriate filters and store the information somewhere.

**Reply by ScotterMonk** [reply]
Thanks. May I ask - within that MessageHandler, what properties access the topic name?

**Reply by Poolitzer** [reply]
[update.message.forum_topic_XXX](https://docs.python-telegram-bot.org/en/v21.3/telegram.message.html#telegram.Message.params.forum_topic_created)
